bite-size-notes/week-4-d-Example of LQR-name.md

```diff
@@ -0,0 +1,350 @@
+$$
+
+$$
+	y(t) = Cx(t) +Du(t) \
+$$
+
+By finding the location of the poles of matrix A, we can understand if the system is stable. We can design a controller that stabilize the system, if the system is unstable, or drive the system to converge faster, if the system is stable. 
+
+By implementing the full state feed back, the closed loop state space representices of the system is:
+
+$$
+\dot{x}(t) = Ax(t)+ Bu(t) \\
+$$
+
+$$
+y(t) = Cx(t) +Du(t) \\
+$$
+
+$$
+u(t) = -Kx(t)
+$$
+
+By sustituting u(t) with -Kx(t), the closed loop state space representices of the system is:
+
+$$ 
+\dot{x}(t) = (A-BK)x(t) \\
+$$
+
+$$
+y(t) = (C-DK)x(t)
+$$
+
+We know that the eigenvalue of the matrix (A-BK) is the poles of the system. We can using K matrix to put the poles at desired location and therefore change the respose of the system. 
+
+
+## How to chose K ?  
+Since we can change the respoce of the system by poles placement. We might ask ourself where should I put the poles, in other words, how to chose the K ? One idea is to construcet the problem into optimization framework that minimize the state error and control nameed LQR.
+
+In class, we already disccused, by using HJB, the optimal control input u^* is a function of A,B,R,Q,P where
+
+$$
+\begin{aligned}
+	A \text{ is the A matrix of the system (know)};\\
+	B \text{ is the B matrix of the system (know)};\\
+	Q \text{ is a states weighing function (know)};\\
+	R \text{ is a control weighing function  (know)};\\
+	P \text{ is a matrix variable form the Riccati equation (Uknow)};
+ \end{aligned}
+$$
+
+And by solving the Ricatti equation, we can find the optimal control input $u^*$
+The variation of the optimal representies of LQR and correponding Riccati equation and control u are summirized below.  
+
+|                                    | LQR form                         | Riccati Equation                |                                
+| ---------------------------------  | -------------------------------- |-------------------------------- |
+| Continuous time finite horizon     | $\min \int_{0}^{T}x(t)^T Q(t) x(t) + u(t)^T R(t) u(t) + x(T)^T Q(T) x(T)$ | $0 =\dot{P}(t) + P(t)A+A^TP(t)+Q-P(t)BR^{-1}B^TP(t)$ | 
+| Discret time finite horizon   | $$\sum_{n=1}^{\infty} x_{k}^T Q_k x_{k} + u_{k}^T R_k u_{k}\$$ | $P_k = Q_k +A^T P_{k+1}A-(A^TP_{k+1}B)(R_k+B^TP_{k+1}B)^{-1}(B^TP_{k+1}A)$ | 
+| Continuous time infinite horizon   | $\min \int_{0}^{\inf}x(t)^T Q(t) x(t) + u(t)^T R(t) u(t)$ |$PA+A^TP+Q-PBR^{-1}B^TP=0$ |
+| Discret time infinite horizon      | $$\min x_{k+1}^T Q_T x_{k+1} + \sum_{k=0}^k x_{k}^T Q_k x_{k} + u_{k}^T R_k u_{k}$$ | $P = Q +A^TPA-(A^TPB)(R+B^TPB)^{-1}(B^TPA)$| 
+
+|                                    |  u          |                     
+| ---------------------------------  | -------------------------------- |
+| Continuous time finite horizon     | $u^*(t) =-R^{-1}B^TP(t)x$ |
+| Discret time finite horizon        | $u^*_k =-(R _{k+1}+B^TP _{k+1}B^{-1}BP _{k+1}A) X _k$  |
```

Comment:
I believe the discrete time finite horizon expression should be $u^*_k =-(R _{k+1}+B^TP _{k+1}B)^{-1}(BP _{k+1}A) x _k$

Reply:
Thanks for correct the equation, I'll fix it